riva/clients/nmt/riva_nmt_t2t_client.cc

```diff
@@ -200,6 +200,7 @@ main(int argc, char** argv)
   if (FLAGS_text != "") {
     nr_nmt::TranslateTextRequest request;
     nr_nmt::TranslateTextResponse response;
+    VLOG(1) << "Setting up t2t config.";
```

Comment:
need this ?

Reply:
I guess this was to fix a build failure. An unrelated build failure is occurring unless some superficial print is added to the nmt module. @sarane22 Is that correct?

Reply:
Yes correct.